scripts/src/java/org/oppia/android/scripts/proto/script_exemptions.proto

```diff
@@ -49,6 +49,11 @@ message TestFileExemptions {
 
       // Overrides the minimum coverage percent required for the given file.
       int32 override_min_coverage_percent_required = 3;
+
+      // A medium-term state to indicate that something about this source file's build config, or
+      // that of its test(s), are not compatible with the current code coverage tooling and thus
+      // should be ignored.
+      bool source_file_is_incompatible_with_code_coverage = 4;
```

Comment:
@Rd4dev can you please file an issue to track fixing the tests exempted per the failures outlined in the PR description, and link that back here? I'll add a code TODO for that issue once it exists.

Reply:
Sure, just filed the tracking issue: #5481